VCFv4.3.tex

```diff
@@ -503,41 +504,87 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
 \label{sv-info-keys}
-The following INFO keys are reserved for encoding structural variants. In general, when these keys are used by imprecise variants, the values should be best estimates. When a key reflects a property of a single alt allele (e.g. SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each alelle (e.g. SVLEN=-100,-110 for a deletion with two distinct alt alleles).
+The following INFO keys are reserved for encoding structural variants. When a key reflects a property of a single alt allele (e.g. SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each alelle (e.g. SVLEN=-100,-110 for a deletion with two distinct alt alleles).
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=IMPRECISE,Number=0,Type=Flag,Description="Imprecise structural variation">
 ##INFO=<ID=NOVEL,Number=0,Type=Flag,Description="Indicates a novel structural variation">
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
 \end{verbatim}
 \normalsize
-For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants the corresponding best estimate.
+For precise variants, END is POS + length of REF allele - 1.
```

Comment:
I'm (in GRIDSS) currently using the IMPRECISE/CIPOS fields to differentiate between variants in which the actual breakpoint position is not known due to lack of coverage, and a variant which the full breakpoint sequence is known but contains a micro-homology thus there is inherit ambiguity in the variant call.

HOMLEN/HOMSEQ don't tell you where the homology is relative to the called position. This information is critical when performing base-pair accurate analysis and the only way to recover it from HOMLEN/HOMSEQ is to recompute it from the reference genome. In the case where the sample differs from the reference, this may not be possible.